vnext/ReactUWP/Base/UwpReactInstance.cpp

```diff
@@ -383,6 +383,56 @@ facebook::react::INativeUIManager* UwpReactInstance::NativeUIManager() const noe
   return m_uiManager->getNativeUIManager();
 }
 
+static std::string PrettyError(const std::string& error) noexcept
+{
+  std::string prettyError = error;
+  if (prettyError.length() > 0 && prettyError[0] == '{')
+  {
+    // if starting with {, assume JSONy
+
+    // Replace escape characters with actuals
+    size_t pos = prettyError.find('\\');
+    while (pos != std::wstring::npos && pos + 2 <= prettyError.length())
```

Comment:
Will it be more efficient to use ostringstream

Reply:
yeah, if this weren't in an error path it would matter more, if we want to improve this even more or make a general utility we can clean that up